Javascript/ants.js

```diff
@@ -34,195 +34,295 @@ var antupgdesc11 = "A valuable offering to the Ant God [Gain up to 3x Sacrifice
 var antupgdesc12 = "Sacrifices itself to allow you to equip [Unlocks the Ant Talisman!]"
 
 const antUpgradeTexts = [null,
-function(){return "ALL Ants work at " + format(Math.pow(1.11 + 1/1000 * player.researches[101], player.antUpgrades[1] + bonusant1),2) + "x speed."},

```

Comment:
These do not need to be functions at all. It is also "unexpected string concatenation" in my ESLint configuration.
These can be changed to use ES6's new method using backticks for strings and pairing them with ${} to display information.

Reply:
After discussions on Discord, we found that function works well, and literal is going to break IE11, so it's also bad.
Yeah, idek why people care about IE nowadays.